neo/bin/api_server.py

```diff
@@ -156,6 +156,8 @@ def main():
     # host
     parser.add_argument("--host", action="store", type=str, help="Hostname ( for example 127.0.0.1)", default="0.0.0.0")
 
+    # rollback
+    parser.add_argument("--rollback", action="store", type=int, help="Block id to rollback the chain to")
```

Comment:
please remove as feature currently doesn't exist

Reply:
Sorry, I thought I have removed everything, seems like that slipped through.